server/models/users.py

```diff
@@ -37,20 +37,17 @@ def updateCredentials(self, credentials):
     # return a user’s data if there is match by email
     @classmethod
     def find_by_email(cls, email):
+        print('finding by email')
         return cls.query.filter_by(email=email).first()
 
-    # return a user’s data if there is match by email
+    # return a user’s data if there is match by id
     @classmethod
     def find_by_id(cls, user_id):
         return cls.query.filter_by(id=user_id).first()
 
     def verify_password(self, password):
         return flask_bcrypt.check_password_hash(self.password, password)
 
-    @classmethod
```

Comment:
We already removed this duplicate function in a previous commit? I think this would remove the the second one?

Reply:
Not sure but i'll work from the newest dev to rebase it all